test/export/test_export.py

```diff
@@ -75,5 +79,34 @@ def forward(self, x):
         # self.assertEqual(mutated_buffer.sum().item(), 30)
         self.assertEqual(output, mod(*inp))
 
+    @config.patch(dynamic_shapes=True, capture_dynamic_output_shape_ops=True, specialize_int=True, capture_scalar_outputs=True)
+    def test_export_constraints(self):
+
+        def f(roi_batch_splits_nms):
+            roi_batch_ids = []
+            for i, x in enumerate(roi_batch_splits_nms.to(torch.int32)):
+                b = x.item()
+
+                add_inline_size_constraint(b, min=0)
```

Comment:
What happens if you call this again, but with different min/max?

Reply:
It'll add both constraints into the graph.

Reply:
Hmm should we add a check to make sure it errors if the second constraint doesn't overlap with the previous one? Is it feasible?

Reply:
Hmm, not sure...it'll error when tracing with dynamo/make_fx though.

Reply:
Ohh hmm can you add a test case where you add conflicting constrains on a symbol and assert that it errors during tracing? We want to make sure the error message is also clear to user

Reply:
done!